platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java

```diff
@@ -458,43 +455,14 @@ public static <K, V> void registerCache(
 
   /**
    * Registers metrics for GuavaCaches using micrometer's GuavaCacheMetrics under the given
-   * cacheName for the cache built using builder and also reports maximum size configured
+   * cacheName for the given guavaCache and also reports maximum size configured
    */
-  public static <K, V> Cache<K, V> registerAndGetCache(
```

Comment:
Do we know if these have been consumed yet?

Reply:
Looks like yes. May want to just deprecate for now then, unless you want to handle tracking down the upgrades preemptively.

Reply:
only one place, api-naming, I will make the changes

Reply:
> Looks like yes. May want to just deprecate for now then, unless you want to handle tracking down the upgrades preemptively.

I'll take care of it, no need to go the deprecated flow. Only two actually, places we have integrated this